Dockerfile

```
@@ -0,0 +1,38 @@
+FROM amazoncorretto:17-alpine AS build
+WORKDIR /workspace/app
+
+COPY gradlew .
+COPY gradle gradle
+COPY build.gradle.kts .
+COPY settings.gradle.kts .
+COPY gradle.lockfile .
+COPY gradle.properties .
+
+COPY eclipse-style.xml eclipse-style.xml
+COPY src src
+COPY api-spec api-spec
```

```suggestion
#COPY api-spec api-spec
```

Comment this section by the moment since this folder is not existent and cause docker build error

---

Fixed with : 773026009b7c83b895129950f87f6d7cb6f9895c